content/_index.md

```diff
@@ -44,6 +44,19 @@ sections:
     color_classes: bg-primary-color-light black
     image: /images/home-page-templating-example.png
     copy: "Hugo's Go-based templating provides just the right amount of logic to build anything from the simple to complex. If you prefer Jade/Pug-like syntax, you can also use Amber, Ace, or any combination of the three."
+sponsors:
+  - name: "Forestry.io"
+    link: https://forestry.io/
+    logo: /images/sponsors/forestry-logotype.svg
+    copy: "Forestry is A full-featured CMS for your Hugo sites"
+  - name: "Netlify"
+    link: https://forestry.io/
+    logo: /images/sponsors/netlify-full-logo-light.svg
+    copy: "Write frontend code. Push it. We handle the rest."
+  - name: "Algolia"
+    link: https://forestry.io/
```

Comment:
Should probably be `https://algolia.com/` 😄

Reply:
That was just dummy content and it's been removed.